k8sfv/leastsquares_test.go

```diff
@@ -17,6 +17,7 @@ package main
 import (
 	. "github.com/onsi/ginkgo"
 	. "github.com/onsi/gomega"
+
```

Comment:
No issue here, but what tool are you running to put these imports into the preferred organization?  I have configuration to run `goimports` whenever I save a file, but it appears that that is not quite sufficient.